python/ray/train/torch/torch_trainer.py

```diff
@@ -141,8 +141,8 @@ def train_loop_per_worker():
 
                     session.report(
                         {},
-                        checkpoint=Checkpoint.from_dict(
-                            dict(epoch=epoch, model=model.state_dict())
+                        checkpoint=TorchCheckpoint.from_state_dict(
+                            model.state_dict()
```

Comment:
Let's keep this as regular Checkpoint until we have an equivalent to store the `epoch` key?